src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,80 @@
+package duke.task;
+
+import java.util.Arrays;
+import java.util.List;
+
+//make abstract class?
```

Comment:
You might want to avoid writing comments as if they are private notes to yourself as stated in the SE textbook [here](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#write-to-the-reader)